### DISCUSSION TITLE
Can I use this library to map groundwater potential zones

### DISCUSSION THREAD

**Discussion by Evanskip31** [pastas/pastas#591] [General]
I'm looking to start mapping groundwater potential zones and I came across this library. Can I use it to map groundwater potential zones? Thank you.

**Comment by raoulcollenteur** [comment]
Hi @Evanskip31 
Thanks for your question. Can you specify a bit what you mean with 'groundwater potential zones' and what variables you want to obtain? My first idea is that other tools like TimML might be more suitable: https://timml.readthedocs.io/en/latest/ for your purpose of modeling.

Cheers,
Raoul

**Reply by Evanskip31** [reply]
Thanks for the prompt reply @raoulcollenteur and thanks for this suggestion.
So I'm thinking of building a local/regional map that is able to show which places have groundwater and what is their depth, and which places don't have or the depth seems to be too far. In the end I would love to simulate a discharge rate and see which regions are likely to be at a risk of water scarcity. What I was first thinking is to have a list of boreholes/wells/springs and their related characteristics (eg. depth of the boreholes, slope and terrain, rainfall, soil, rocks etc.), then from which I could train the data and later on build a prediction model to predict those other places if they are likely to have groundwater in them or not.

Many thanks,
Evans.
